spec/hyacinth/adapters/digital_object_search_adapter/solr_spec.rb

```diff
@@ -10,10 +10,50 @@
 
   context "#solr_params_for" do
     let(:solr_params) { adapter.solr_params_for(search_params) }
-    context "with multiple filter values on the same filter" do
-      let(:search_params) { { 'animals' => ['dogs', 'cats'] } }
+    context "with multiple filters on the same field" do
+      let(:search_params) do
+        {
+          'animals' => [
+            ['aardvarks', 'DOES_NOT_EXIST'], ['celocanths', 'CONTAINS'], ['elephants', 'EQUALS'],
```

Comment:
`['aardvarks', 'DOES_NOT_EXIST']` and `['panthers', 'EXISTS']`

I wasn't expecting to see any search values associated with these match types.

Reply:
Is this test meant to demonstrate that if you supply a value that it will be ignored by these two match types?  This might be confusing for someone looking at the tests as usage examples.